spynnaker/pyNN/models/neuron/neuron_models/neuron_model_izh.py

```diff
@@ -78,7 +78,7 @@ def has_variable(self, variable):
     @overrides(AbstractNeuronModel.get_global_values,
                additional_arguments={'machine_time_step'})
     def get_global_values(self, machine_time_step):
-        return [machine_time_step]
+        return [machine_time_step/1000]
```

Comment:
This should probably be float(machine_time_step) / 1000.0, as machine_time_step might be < 1000

Reply:
should also remove magic number and replace with a named constant. Had a little look around our constants. Seems we're missing that constant. so likely needs a few places to change